### PR TITLE
Request edge metrics only if prometheus metrics data is empty

### PR DESCRIPTION
**Pull request by zhu733756**
Signed-off-by: zhu733756 <talonzhu@yunify.com>

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Request edge metrics only if prometheus metrics data is empty. Currently, it requests the edge metrics everytime.

### CODE REVIEW

File: pkg/models/monitoring/monitoring.go [hunk 1]
```diff
@@ -115,18 +115,27 @@ func (mo monitoringOperator) GetNamedMetrics(metrics []string, time time.Time, o
...
 		edgeMetrics := make(map[string]monitoring.MetricData)
-		for _, metric := range mr {
-			edgeMetrics[metric.MetricName] = metric.MetricData
-		}
 
-		for i, metric := range ress {
-			if val, ok := edgeMetrics[metric.MetricName]; ok {
-				ress[i].MetricData.MetricValues = append(ress[i].MetricData.MetricValues, val.MetricValues...)
+		for i, ressMetric := range ress {
+			metircName := ressMetric.MetricName
```
Comment (benjaminhuo) [inline pkg/models/monitoring/monitoring.go:123]: metircName => metricsName

File: pkg/models/monitoring/monitoring.go [hunk 2]
```diff
@@ -137,17 +146,25 @@ func (mo monitoringOperator) GetNamedMetricsOverTime(metrics []string, start, en
...
```
Comment (benjaminhuo) [inline pkg/models/monitoring/monitoring.go:154]: metircName => metricName

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: pkg/models/monitoring/monitoring.go (hunk 1 above)
Comment (zhu733756) [reply pkg/models/monitoring/monitoring.go]: fixed

File: pkg/models/monitoring/monitoring.go (hunk 2 above)
Comment (zhu733756) [reply pkg/models/monitoring/monitoring.go]: fixed

**Review by benjaminhuo — APPROVED** [review]
/lgtm

**Pull request merged**